docs/process/Development.md

```diff
@@ -17,10 +17,12 @@ You are responsible for shepherding your work through the development process. I
 > Issue transitions to **In Review**
 
 - Notify other devs and wait for reviews
+- Add the **needs test** label
```

Comment:
try to keep your language consistent with the rest of the doc. elsewhere it says `Label your PR **ready**`, so either update yours to match, or change that.